cryptoorgchain/assetlist.json

```diff
@@ -3,7 +3,7 @@
     "chain_name": "cryptoorgchain",
     "assets": [
       {
-        "description": "CRO coin is the token for the Crypto.com platform.",
+        "description": "Crypto.org Chain - driving mass adoption of blockchain technology through use cases like Payments, DeFi and NFTs, prioritizing speed and seamless transactions.",
```

Comment:
This description should be reserved for the CRO asset. This kind of description can go under chain.json

Reply:
Noted, we will revise it. Thank you.